apps/gateway/test/champions_test.exs

```diff
@@ -295,13 +280,134 @@ defmodule Gateway.Test.Champions do
     end
   end
 
-  defp get_last_message() do
-    receive do
-      message ->
-        message
-    after
-      5000 ->
-        raise "No message"
+  describe "campaigns" do
+    test "get campaigns and levels", %{socket_tester: socket_tester} do
+      # Register user
+      {:ok, user} = Users.register("campaign_user")
+
+      # GetCampaigns
+      SocketTester.get_campaigns(socket_tester, user.id)
+      fetch_last_message(socket_tester)
+
+      assert_receive %WebSocketResponse{
+        response_type: {:campaigns, %Campaigns{} = campaigns}
+      }
+
+      assert Enum.count(campaigns.campaigns) > 0
```

Comment:
I think comparing that there's any campaign is not enough, if there's not archiving of soft-delete mechanism we could do a `GameBackend.Repo.all(GameBackend.Campaigns)`  and check if the numbers match to ensure that we're retrieving the campaigns correctly